### DISCUSSION TITLE
Support lazy loading

### DISCUSSION THREAD

**Discussion by 4x4notfound** [family/connectkit#98] [Ideas]
Startup times in restrictive environments like cloudflare workers is too slow and adds about 700kb gzipped

**Comment by lochie** [comment]
Great idea @maurerbot, we'll look into this.
But not sure why that's appearing so high when our package is only 462kb unzipped. That size probably accounts for wagmi and ethers as well?

**Reply by 4x4notfound** [reply]
yes, exactly. the co-deps get it closer to 2000kb

**Reply by 4x4notfound** [reply]
you can probably close this cause I was able to get it working: https://github.com/kubelt/kubelt/blob/main/apps/passport/app/components/connect-button/ConnectButton.tsx

This also has the benefit of reducing bundle size.
